packages/google_maps_flutter/google_maps_flutter/android/src/main/java/io/flutter/plugins/googlemaps/GoogleMapController.java

```diff
@@ -795,6 +818,18 @@ private void updateInitialCircles() {
     circlesController.addCircles(initialCircles);
   }
 
+  @Override
+  public void setInitialTileOverlays(Object initialTileOverlays) {
```

Comment:
Can we make the method takes `List<Object> initialTileOverlays` as the parameter directly.

Reply:
I did in the same style as the existing code